debian/patches/cpick-7a0265d3-fix-ensure-MAAS-datasource-retries-on-failure-6167

```diff
@@ -0,0 +1,101 @@
+From 7a0265d36e01e649f72005548f17dca9ac0150ad Mon Sep 17 00:00:00 2001
+From: James Falcon <james.falcon@canonical.com>
+Date: Wed, 16 Apr 2025 08:35:15 -0500
+Subject: [PATCH] fix: ensure MAAS datasource retries on failure (#6167)
+
+All MAAS requests get an `exception_cb` added dynamically via the
+`OauthUrlHelper`. Ensure these callbacks adhere to the requirement
+added in 1a515532 that an `exception_cb` return True if the exception
+is not to be raised from the handler.
+
+LP: #2106671
+---
```

Comment:
Per the suggestions from quilt about this patch:
```
If this cherry-pick is due to a bug to be SRU'ed, also add to the DEP-3 header:
    Bug: <reference to GH issue>
    Bug-Ubuntu: <reference to LP issue>
   ```

Let's add the DEP-3 structured references too as footers below the multi-line subject
```suggestion
---
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/cloud-init/+bug/2106671
```